internal/cri/server/restart.go

```diff
@@ -174,7 +174,13 @@ func (c *criService) recover(ctx context.Context) error {
 					WithError(err).
 					WithField("container", container.ID()).
 					Error("Failed to load container")
-
+				if _, ok := err.(containerstore.StoreStatusError); ok {
+					return err
```

Comment:
returning err here will cause the error group with context to cancel all sub tasks in this container load loop... shouldn't we just return nil on load fail and continue the image and cleanup after all the containers that can load are loaded?